Project_One/about.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<body id="grading-element-two">
+    <button id = "grading-element-one" onclick="addit()">Don't Click!!</button>
+    <script>
+function addit() {
+    var add = document.createElement("img")
+    add.src ="Photos/DontClick.jpg"
+    add.setAttribute("id", "grading-element-one-consequence");
```

Comment:
can you make it `grading-element-two-consequence`, after moving a `grading-element-one-consequence` to `index.html`?